### ISSUE TITLE
Normal (Gaussian Curve) formula breaks with Tensorflow 1.01

### ISSUE THREAD

**Issue by YanYas** [pkmital/CADL#66]
Hi, I was trying out the tutorials on windows and I found that the gaussian formula wasn't working. Something to do with the dtypes not converting correctly. None the less, there is a solution in the form of the Normal Distribution [contribution](https://www.tensorflow.org/api_docs/python/tf/contrib/distributions/Normal#mu) in the update. I dunno if you want to re-record that bit though. I'll see if I can figure out how it works later and mention it in the Kadenze forum.

**Comment by pkmital** [comment]
Hi and thanks!  Good to note the https://www.tensorflow.org/api_docs/python/tf/contrib/distributions/Normal distribution.  Is there any specific error on windows?  I will try to take a look at it then!  Thanks

**Comment by YanYas** [comment]
I'm not in front of my machine at the moment but I don't think the issue
strictly a Windows one. I saw a similar error elsewhere on Stack Overflow
and it referred to a change in the api.

I think it might be worth also having a look at tf.nn.moments and
tf.nn.batchnormalization, although I found it important to see the practice
broken down into explicit parts so I could understand what the functions
were doing. They could be employed in later tutorials where students
already understand what they do to speed up the process of setting up a
graph.

Anyways, I'll take a look at the error and post it to you.
